web-ts/src/components/__tests__/PoiListItem.spec.tsx

```diff
@@ -1,5 +1,3 @@
-// @flow
```

Comment:
This would basically be almost no work to rewrite as without enzyme and snapshot. Would you mind? :)